doc/README.md

```diff
@@ -238,6 +239,15 @@ grid.on('resizestart', function(event: Event, el: GridItemHTMLElement) {
 });
 ```
 
+### resize(event, el)
+
+called while the user is resizing an item, with current node attributes
+
+```js
+grid.on('resize', function(event: Event, item: GridStackNode) {
```

Comment:
don't pass the node, pass the element (from which node can be accessed) to be consistent... then they can use the actual px width/height or node x,y (column based)